source/math/FloatMatrix.ooc

```diff
@@ -45,6 +45,36 @@ FloatMatrix : cover {
 		result
 	}
 
+	setColumn: func (x: Int, values: ...) {
```

Comment:
Currently the only potential use of this (the only current use FloatMatrix in general?) I can find involves 2x2 and 2x1 matrices.